src/librustdoc/html/markdown.rs

```diff
@@ -26,13 +26,10 @@
 
```

Comment:
I can't tell whether or not the documentation for this is rendered anywhere (?), but it seems like this comment should be updated, regardless.  It talks about the "hoedown C-library" and "the C bindings."  Also, the example might not work anymore, it uses `Markdown(s)`, but that appears to now be `Markdown(s, style)`